tests/test_computeExposureSummaryStats.py

```diff
@@ -101,12 +101,51 @@ def testComputeExposureSummary(self):
         background.append(backobj)
 
         # Configure and run the task
+        expSummaryTaskNoUpdates = ComputeExposureSummaryStatsTask()
         expSummaryTask = ComputeExposureSummaryStatsTask()
         # Configure nominal values for effective time calculation
         expSummaryTask.config.fiducialZeroPoint = {band: float(zp)}
         expSummaryTask.config.fiducialPsfSigma = {band: float(psfSize)}
         expSummaryTask.config.fiducialSkyBackground = {band: float(skyMean)}
-        # Run the task
+
+        # Run the task with optianal updates turned off
+        expSummaryTaskNoUpdates.config.doUpdatePsfModelStats = False
+        expSummaryTaskNoUpdates.config.doUpdateApCorrModelStats = False
+        expSummaryTaskNoUpdates.config.doUpdateMaxDistToNearestPsfStats = False
+        expSummaryTaskNoUpdates.config.doUpdateWcsStats = False
+        expSummaryTaskNoUpdates.config.doUpdatePhotoCalibStats = False
+        expSummaryTaskNoUpdates.config.doUpdateBackgroundStats = False
+        expSummaryTaskNoUpdates.config.doUpdateMaskedImageStats = False
+        expSummaryTaskNoUpdates.config.doUpdateEffectiveTimeStats = False
+
+        summary = expSummaryTaskNoUpdates.run(exposure, None, background)
+        # Test the outputs
+        nan = float("nan")
+        self.assertTrue(summary.ra, nan)
+        self.assertTrue(summary.dec, nan)
```

Comment:
`assertTrue`? I don't think this is doing what you think it is:

https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertTrue

Did you mean `self.assertTrue(np.isnan(summary.ra))`, and similarly for all the tests below? Note that you cannot test equality for NaN, because it's not equal to itself.

Reply:
Thanks for catching that!